test/unit/injector-deployment.bats

```diff
@@ -363,6 +363,87 @@ load _helpers
   [ "${value}" = "false" ]
 }
 
+#--------------------------------------------------------------------
+# securityContext or pod and container
+
+@test "injector/deployment: default pod securityContext" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      --show-only templates/injector-deployment.yaml  \
+      . | tee /dev/stderr |
+      yq -r '.spec.template.spec.securityContext' | tee /dev/stderr)
+  [ "${actual}" = "null" ]
+}
+
+@test "injector/deployment: custom pod securityContext" {
```

Comment:
Can you change these tests to check something besides the default values? Doesn't need to be too exhaustive, just overriding a couple of the options would suffice.

Reply:
added some other values